docs/guides/color-modes.md

```diff
@@ -7,7 +7,7 @@ Usually a dark mode feature includes changes to the colors of a site without cha
 This guide will walk through one approach (the one used on this site) that includes multiple color modes that can be changed by the end user.
 
 Since the only styles that change between these color modes are the colors themselves, the different color palettes are stored in the `theme.colors` object.
-If you look at this site's [`src/theme.js`][] file, you'll see that it includes a nested `colors.modes` object for the different color schemes.
+If you look at this site's [`src/gatsby-plugin-theme-ui/index.js`][] file, you'll see that it includes a nested `colors.modes` object for the different color schemes.
```

Comment:
I think you forgot to update the link this references below

Reply:
🙄 ah, yes, sorry, I misunderstood the markdown syntax, lazy PR'ing.

Do you want me to update the text, so it reads something like `If you look at this site's theme file`? The updated filepath/name is less clear it's the main theme file, without going into questions of where gatsby fits into it all. Overthinking this perhaps. :)

Reply:
No worries! Yeah, if you wanna update the language here, I think that sounds fine

Reply:
:+1: